libdevcore/CommonData.h

```diff
@@ -255,6 +255,12 @@ bool contains(T const& _t, V const& _v)
 	return std::end(_t) != std::find(std::begin(_t), std::end(_t), _v);
 }
 
+template <class T, class Predicate>
+bool contains_if(T const& _t, Predicate const& _p)
```

Comment:
That's pretty much ``std::count_if``, isn't it? Resp. the ranged version of that, which ``boost::count_if`` has.

Reply:
with the difference that it stops once it found one.. just like `contain` a few lines up

Reply:
Ok!